schemas/maas-backend/customers/customer.json

```diff
@@ -31,6 +34,12 @@
       "items": {
         "$ref": "http://maasglobal.com/core/components/authorization.json"
       }
+    },
+    "favoriteLocations": {
+      "type": "array",
+      "items": {
+        "type": "object"
+      }
     }
   },
   "required": ["personalData", "paymentSources", "balances", "region", "authorizations"],
```

Comment:
Are some of the fields required?

Reply:
I think "identityId" is required, but i'm not certain, so to prevent breaking changes, i'll leave it for now.
The fix is done only to ensure "identityId" is not additionalProperty